cardano-cli/src/Cardano/CLI/Shelley/Orphans.hs

```diff
@@ -58,9 +59,9 @@ deriving newtype instance ToJSON ByronHash
 -- This instance is temporarily duplicated in cardano-config
 
 instance ToJSON (HeaderHash blk) => ToJSON (Tip blk) where
-  toJSON TipGenesis = object [ "genesis" .= True ]
+  toJSON TipGenesis = J.object [ "genesis" .= True ]
```

Comment:
Why the qualified import?  If you are going to qualify aeson imports we do it via `Aeson`. In this case I think its overkill to qualify `object` as it is fairly obvious already that it comes from the aeson library.

Reply:
I've switched it to `Aeson`.  I'm trying to avoid unqualified imports and at the same time keep the qualified import short so it doesn't wrap.